torch/distributed/distributed_c10d.py

```diff
@@ -1396,6 +1396,7 @@ def wrapper(*args, **kwargs):
                     "func_name": f"{func.__name__}",
                     "args": f"{args}, {kwargs}",
                     "backend": f"{get_backend()}",
+                    "device": f"{_get_pg_device(kwargs.get('group'))}",
```

Comment:
FYI, `_get_pg_device` may be deprecated or changed due to the changes in https://github.com/pytorch/pytorch/pull/88330 which convert all PG-specifics instances to type Backend

Reply:
Thanks for the info. Will wait for the change then.